__tests__/app.test.js

```diff
@@ -55,31 +56,82 @@ describe('ERROR GET /api/topicd', () => {
 describe('GET /api/articles/:article_id', () => {
   test('sends GET to ID endpoint -> returns an object', () => {
     return request(app)
-    .get('/api/articles/1')
-    .expect(200)
-    .then((res) => {
-      const output = res.body.rows[0];
-      expect(output).toBeInstanceOf(Object);
-      expect(Object.keys(output).length).toBe(7);
+      .get('/api/articles/1')
+      .expect(200)
+      .then((res) => {
+        const output = res.body.rows[0];
+        expect(output).toBeInstanceOf(Object);
+        expect(Object.keys(output).length).toBe(7);
     });   
   });
 
   test('sends GET to ID endpoint -> checks each object key', () => {
     return request(app)
-    .get('/api/articles/1')
-    .expect(200)
-    .then((res) => {
-      const output = res.body.rows[0];
-      expect(output).toHaveProperty('author');
-      expect(output).toHaveProperty('title');
-      expect(output).toHaveProperty('article_id');
-      expect(output).toHaveProperty('body');
-      expect(output).toHaveProperty('topic');
-      expect(output).toHaveProperty('created_at');
-      expect(output).toHaveProperty('votes');
-    });   
+      .get('/api/articles/1')
+      .expect(200)
+      .then((res) => {
+        const output = res.body.rows[0];
+        expect(output).toHaveProperty('author');
+        expect(output).toHaveProperty('title');
+        expect(output).toHaveProperty('article_id');
+        expect(output).toHaveProperty('body');
+        expect(output).toHaveProperty('topic');
+        expect(output).toHaveProperty('created_at');
+        expect(output).toHaveProperty('votes');
+      });   
   });
 });
 
+describe('PATCH /api/articles/:article_id', () => {
+  test('sends PATCH to ID endpoint -> checks each object key', () => {
+    return request(app)
+      .patch('/api/articles/1')
+      .send({ inc_votes : 1 })
+      .expect(201)
+      .then((res) => {
+        const output = res.body.rows[0];
+        expect(output).toBeInstanceOf(Object);
+        expect(Object.keys(output).length).toBe(7);
```

Comment:
if we are sending 1 vote, what is the total vote count for the article after is been sent?

Reply:
please see line 120. Also 130 which tests a negative vote's value. Just I opted to place these in separate tests because I recall it being said that we should be careful when making many assertions in one test.